app/__tests__/services/media.test.ts

```
@@ -86,106 +236,713 @@
+      expect(mockAudioElement.setSinkId?.mock.calls[0][0]).toBe('output2');
+    });
+
+    test('should check if setSinkId is supported', () => {
+      // With support
+      const supported = mediaManager.isSinkIdSupported(
+        mockAudioElement as unknown as HTMLMediaElement
+      );
+      expect(supported).toBe(true);
+
+      // Without support
+      delete mockAudioElement.setSinkId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)